libraries/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -194,6 +194,11 @@
                 "type": "string"
               }
             },
+            "disableBuildCache": {
```

Comment:
This doesn't appear to be referenced anywhere.

Reply:
It's a bit sneaky, the raw command object gets `...`ed into the parsed result, and a handful of fields get reprocessed. The `disableBuildCache` property is one of the ones picked up by that initial pass.